app/models/api/v3/readonly/dashboards/country.rb

```diff
@@ -9,15 +9,6 @@
 #  commodity_id  :integer
 #  node_id       :integer
 #
-# Indexes
```

Comment:
Any idea why this disappeared? I think in the migration you're recreating the indexes correctly.